lib/credo/check/warning/dbg.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Credo.Check.Warning.Dbg do
+  use Credo.Check,
+    base_priority: :high,
```

Comment:
We could think about deactivating this check for older versions of Elixir:

```suggestion
    base_priority: :high,
    elixir_version: ">= 1.14.0-dev",
```